src/components/JoinModal/JoinModal.js

```diff
@@ -77,83 +100,105 @@ const JoinModal = () => {
 
   const openModal = () => {
     setIsModalOpen(true);
+    document.body.style.overflow = 'hidden';
   };
   return (
-    <>
-      <GoToJoin onClick={openModal}>소인 그대와 조인할것이옵니다.</GoToJoin>
-      {isModalOpen && (
-        <Modal>
-          <ModalForm>
-            <ExitBtn type="button" onClick={goToDetail}>
-              ☓
-            </ExitBtn>
-            <ProjectName>
-              <ProjectNameText>프로젝트명 : </ProjectNameText>
-              <ProjectNameText>남은 인원 : </ProjectNameText>
-              <ProjectNameText>필수 스택 : </ProjectNameText>
-            </ProjectName>
-            <CheckBoxWrap>
-              <Title noMargin="noMargin">* Position</Title>
-              <CheckBox>
-                <PositionBtn
-                  type="radio"
-                  id="checkBox"
-                  name="position"
+    metaData.positions && (
```

Comment:
이 부분도 early return해주는 게 더 좋을 것 같습니다.

```suggestion
    if(!metaData.positions) return <>데이터 없음!</>
    
    return (
       <GoToJoin ...>
```